### ISSUE TITLE
Establish critical bugs board and process

### ISSUE THREAD

**Issue by cbeams** [bisq-network/projects#16]
Board: https://github.com/orgs/bisq-network/projects/11

- [x] @ripcurlx to list current set of known bugs at https://docs.google.com/spreadsheets/d/199UV61b3A8QylMmYZMw-FT79N9VgKRvYAwC9pF4TWE0/edit#gid=0
 - [x] @cbeams to create [Critical Bugs board](https://github.com/orgs/bisq-network/projects/7)
   - [x] @cbeams to add a few items from the spreadsheet to the board to get things started
 - [x] @ripcurlx to go through spreadsheet and add critical bugs to the new board (bisq-network/admin#33)
 - [x] @ripcurlx to transcribe any notes and linked issues from the spreadsheet to the actual critical bug issues (bisq-network/admin#33)
 - [x] @leo816 to work with @ripcurlx to make sure critical bugs listed in private support spreadsheet are reflected on the new board
 - [x] @cbeams to stub out documenting the Critical Bugs board and process on the wiki
   - [x] @ripcurlx (has been asked) to complete it
 - [x] @cbeams (@ripcurlx?) to announce the board and process to bisq-network/dev (or all of bisq-network/dao?)
 - [x] Link to the critical bugs wiki article from new developer entry points, including:
   - [x] CONTRIBUTING.md in the main repo (bisq-network/bisq#3996)
   - [x] https://docs.bisq.network/contributor-checklist.html (bisq-network/bisq-docs#193)
   - [x] https://bisq.wiki/Development_Process page
 - [x] @cbeams to create https://bisq.wiki/Bisq_Maintainer role documentation and add link to Critical Bugs process (bisq-network/admin#39)

**Comment by ripcurlx** [comment]
* [x] @ripcurlx to go through spreadsheet and add critical bugs to the new board
* [x]  @ripcurlx to transcribe any notes and linked issues from the spreadsheet to the actual critical bug

**Comment by leo816** [comment]
- [x] @leo816 to work with @ripcurlx to make sure critical bugs listed in private support spreadsheet are reflected on the new board

**Comment by cbeams** [comment]
@ripcurlx, now that the Critical Bugs board is in place and populated, we're getting close to this project being complete. What remains is to document and announce the board and its process. To that end, I've created a stub at https://bisq.wiki/Critical_Bugs. Would you be willing to flesh it out? If so, please create an admin task, assign it to yourself and reference it here, thanks.

You'll see that I've also created stubs for the Dev Team and the larger Development Process of which the Critical Bugs board/process are a part. Wiki links should connect all of them (and mostly already do).

**Comment by ripcurlx** [comment]
- [x] Updated wiki page for critical bugs: https://bisq.wiki/Critical_Bugs

**Comment by ripcurlx** [comment]
- [x] announced critical bugs board in GitHub dev team as discussion: https://github.com/orgs/bisq-network/teams/dev/discussions/2

**Comment by ripcurlx** [comment]
@cbeams Do you want to close this issue after your wiki edits?

**Comment by cbeams** [comment]
yes, will do.

**Comment by cbeams** [comment]
> @cbeams Do you want to close this issue after your wiki edits?
>> yes, will do

Actually, this project shouldn't be considered complete until the board and process are linked to from the places that (especially new) developers would look to as a reference on our development process. I've added checkboxes in the description to this effect. I would be fine if these links are surgically added in a rough / quick way. We should more fully revise and unify the resources mentioned there, but that's a separate effort.

As an aside, this is a good example of end-to-end thinking when it comes to projects / what it means to really deliver something. We created the board, documented the process and announced it to `@bisq-network/dev` and `@bisq-network/support`, but without the links mentioned above, it's likely that future developers won't know about the board/process unless someone mentions it to them.

**Comment by ripcurlx** [comment]
- [x] https://bisq.wiki/Development_Process
- [x] https://docs.bisq.network/contributor-checklist.html 
  - [x] Create PR: https://github.com/bisq-network/bisq-docs/pull/193
- [x] CONTRIBUTING.md in the main repo
  - [x] Create PR: https://github.com/bisq-network/bisq/pull/3996

**Comment by cbeams** [comment]
Ok, I've updated the description to reflect recent proliferation of links to the critical bugs article in all the right places. We're just waiting on the two PRs to be merged, and I think we can consider this project delivered.

**Comment by ripcurlx** [comment]
> Ok, I've updated the description to reflect recent proliferation of links to the critical bugs article in all the right places. We're just waiting on the two PRs to be merged, and I think we can consider this project delivered.

Both PRs were merged: Seems to be a very active maintainer 😉

**Comment by cbeams** [comment]
Please actually check boxes in the description when you complete them, thanks. It's also OK to re-post them as comments as has been done above here, but the only essential thing is to check the boxes in the description and add a comment saying you've done so. Thanks.

**Comment by cbeams** [comment]
I just want to note here that this is our first delivered project under the new proposed project management infrastructure at #1. When we put together the first review meeting, we can go over this as an example.

I've just labeled the project as `was:delivered` accordingly.
